src/sc_keeper/api.py

```diff
@@ -168,7 +179,7 @@ def search_server(
         Query(
             title="Maximum price",
             description="Maximum price (USD/hr).",
-            json_schema_extra={"category_id": FilterCategories.PRICE, "step": 0.0001},
+            json_schema_extra={"category_id": FilterCategories.PRICE, "step": 0.0001, "unit": "USD"},
```

Comment:
hm, this will be super confusing when someone is passing the `currency` field. IMO we should query the conversion rate between the queried `currency` and USD and pass the multiplication if those in SQL so that the price field is actually in the queried currency, instead of static USD